### ISSUE TITLE
Security warning when installing on a Mac

### ISSUE THREAD

**Issue by vladstudio** [drovp/drovp#5]
You probably already know that, but MacOS does not allow me to install the app, suggesting I move it to Bin. I have to open Security preferences and explicitly allow "this developer" apps to be installed. 

I suspect it's related to $99/year account, so no hard feelings ;-)

**Comment by tomasklaen** [comment]
Yeah I don't have an apple dev account yet. This is in development mode atm, and since the app targets power users I though this won't be a problem.

I will sort it out eventually as (or if) this gains more users and traction.
